tests/apichecks/crds_test.go

```diff
@@ -233,6 +233,93 @@ func TestCRDsAcronyms(t *testing.T) {
 	test.CompareGoldenFile(t, "testdata/exceptions/acronyms.txt", want)
 }
 
+// Avoid passing sensitive data as plain text in the CRD
+func TestNoSensitiveField(t *testing.T) {
+	crds, err := crdloader.LoadAllCRDs()
+	if err != nil {
+		t.Fatalf("error loading crds: %v", err)
+	}
+
+	var errs []string
+
+	sensitiveKeywords := []string{
+		"password",
+	}
+	filter := func(fieldPath string) bool {
```

Comment:
nit: recommend using a function name more specific to its functionality, something like "hasSensitiveSuffix" or "endWithSensitiveKeywords"?

Reply:
Used `sets.Set` as Justin suggested. It looks much nicer. Could you take another look.